package.json

```
@@ -25,7 +25,7 @@
     "coverage:fast": "xvfb-maybe cross-env INTEGRATION_TESTS=0 TS_NODE_PROJECT='./tsconfig.test.json' TS_NODE_FILES=1 nyc mocha './tools/test-globber.ts' && nyc report --reporter=text-lcov > coverage.lcov",
     "coverage:slow": "xvfb-maybe cross-env TS_NODE_PROJECT='./tsconfig.test.json' TS_NODE_FILES=1 nyc mocha './tools/test-globber.ts' --integration && nyc report --reporter=text-lcov > coverage.lcov",
     "docs": "bolt build:full && bolt docs:generate",
-    "docs:generate": "ts-node ./tools/gen-docs.ts",
+    "docs:generate": "node --max-old-space-size=8192 -- node_modules/.bin/ts-node ./tools/gen-docs.ts",
```

any reason not to use one of the recommended ways of passing flags through ts-node to node here? see: https://typestrong.org/ts-node/docs/configuration/#node-flags

```suggestion
    "docs:generate": "node --max-old-space-size=8192 -r ts-node/register ./tools/gen-docs.ts",
```

---

i just copy-pasted from search engine 🙈 https://github.com/TypeStrong/ts-node/issues/261#issuecomment-569035006